.travis.yml

```diff
@@ -103,7 +105,7 @@ matrix:
         - os: linux
           env: PYTHON_VERSION=3.5 SETUP_CMD='test --remote-data=astropy -a "--mpl"'
                CONDA_DEPENDENCIES=$CONDA_ALL_DEPENDENCIES
-               PIP_DEPENDENCIES='jplephem pytest-mpl'
+               PIP_DEPENDENCIES="jplephem pytest-mpl $ASDF_GIT"
```

Comment:
Maybe I should add this dependency to the test on Py3.6 as well? Or maybe just that one instead of this one?

Reply:
either case sounds good to me.

Reply:
I realized it at least has to be in the coverage test or else the coverage metrics get all messed up. I think I'll leave it in both, though.